pkg/storage/imagestore/imagestore.go

```diff
@@ -194,6 +197,14 @@ func (is *ImageStore) initRepo(name string) error {
 
 			return err
 		}
+
+		if is.events != nil {
+			if err := is.events.RepositoryCreated(name); err != nil {
+				is.log.Error().Err(err).Str("name", name).Msg("failed to emit event")
+
+				return err
```

Comment:
Do we want the calls to image store to error in case there are errors emitting events? Would we want the image push to fail (propagate to client side) in case there are events related errors?

The same comment applies to error handling in the other functions in this file.

Reply:
Yeah I agree we should log and swallow the errors in the image store.